osu.Game.Tournament/JsonPointConverter.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Diagnostics;
+using System.Drawing;
+using Newtonsoft.Json;
+
+namespace osu.Game.Tournament
+{
+    /// <summary>
+    /// We made a change from using SixLabors.ImageSharp.Point to System.Drawing.Point at some stage.
+    /// This handles converting to a standardised format on json serialize/deserialize operations.
+    /// </summary>
+    internal class JsonPointConverter : JsonConverter<Point>
+    {
+        public override void WriteJson(JsonWriter writer, Point value, JsonSerializer serializer)
+        {
+            // use the format of LaborSharp's Point since it is nicer.
+            serializer.Serialize(writer, new SixLabors.ImageSharp.Point(value.X, value.Y));
+        }
+
+        public override Point ReadJson(JsonReader reader, Type objectType, Point existingValue, bool hasExistingValue, JsonSerializer serializer)
+        {
+            if (reader.TokenType != JsonToken.StartObject)
+            {
+                // if there's no object present then this is using string representation (System.Drawing.Point serializes to "x,y")
+                string str = (string)reader.Value;
+
+                Debug.Assert(str != null);
+
+                var split = str.Split(',');
+
+                return new Point(int.Parse(split[0]), int.Parse(split[1]));
```

Comment:
It is slightly safer to use `System.Drawing.PointConverter` for this. This will work for 99.9% of cases but not the potential 0.01% (like a system with a weird culture setting that doesn't use commas for [list separators](https://docs.microsoft.com/pl-pl/dotnet/api/system.globalization.textinfo.listseparator?view=net-5.0)).